src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -1521,6 +1522,584 @@ public void canDeleteARequest()
       getResponse.getStatusCode(), is(HttpURLConnection.HTTP_NOT_FOUND));
   }
 
+  @Test
+  public void canExpireASingleOpenUnfilledRequest()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    UUID id = UUID.randomUUID();
+    UUID itemId = UUID.randomUUID();
+
+    JsonObject requestRequest = new RequestRequestBuilder()
+      .hold()
+      .withId(id)
+      .withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54, DateTimeZone.UTC))
+      .withItemId(itemId)
+      .withPosition(1)
+      .withStatus(OPEN_NOT_YET_FILLED)
+      .create();
+    CompletableFuture<JsonResponse> createCompleted = new CompletableFuture<>();
+    client.put(requestStorageUrl(String.format("/%s", id)),
+      requestRequest, StorageTestSuite.TENANT_ID,
```

Comment:
I think most of the creation logic can be replaced with the `createRequest` method.

```suggestion
      createRequest(requestRequest);
```

Reply:
replaced with createEntity (moved createRequest to superclass and renamed to createEntity to make it reusable)